modules/kubernetes-addons/helm-addon/outputs.tf

```diff
@@ -0,0 +1,10 @@
+output "helm_release_addon" {
+  description = "Describes the desired status of a chart in a kubernetes cluster."
+  value = try(
+    {
+      for k, v in helm_release.external-dns : k => (
```

Comment:
```suggestion
      for k, v in helm_release.addon : k => (
```

Reply:
That's what I get for copy pasting..
Thanks.